tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -2204,4 +2172,19 @@ public async Task MockDirectory_Exists_ShouldReturnTrue_IfArgIsFrontSlashAndRoot
 
         await That(fileSystem.Directory.Exists("/")).IsEqualTo(true);
     }
+
+    [Test]
+    public async Task MockDirectory_Move_ShoudlNotThrowException_When_SourceAndDestLocationAreSame()
```

Comment:
```suggestion
    public async Task MockDirectory_Move_ShouldNotThrowException_When_SourceAndDestinationDifferOnlyInCasing()
```

Issue #1138 is about not throwing an exception on windows, when source and destination only differ in casing. The other behavior should remain unchanged!

Reply:
Corrected on latest commits.